docs/src/pages/components/lists/AlignItemsList.tsx

```diff
@@ -11,8 +11,6 @@ import Typography from '@material-ui/core/Typography';
 const useStyles = makeStyles((theme: Theme) =>
   createStyles({
     root: {
-      width: '100%',
-      maxWidth: 360,
```

Comment:
It breaks the purpose of the demo:

> You should change the list item alignment when displaying 3 lines or more, set the alignItems="flex-start" property.

Reply:
I don't understand the purpose from this sentence. How does the width relate to the number of items?

Reply:
The idea is to have list item with enough height to show the impact of the flex positioning of the avatar (top vs middle)

Reply:
So why not set the height instead?

Reply:
Relying on text wrapping avoids the need to know the computed line-height. In our case, I think that a width ch approach would work.